packages/web-components/src/components/masthead/megamenu.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, customElement, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import StableSelectorMixin from '../../globals/mixins/stable-selector';
+import styles from './masthead.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * MegaMenu
+ *
+ * @element dds-megamenu
+ * @slot brand - The left hand area.
+ * @slot nav - The nav content.
+ * @slot profile - The right hand area.
+ */
+@customElement(`${ddsPrefix}-megamenu`)
+class DDSMasthead extends StableSelectorMixin(LitElement) {
+  render() {
+    return html`
+      <section tabindex="-1" class="${prefix}--masthead__megamenu">
```

Comment:
Please see: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/web-components/docs/coding-conventions.md#custom-element-itself-as-an-eleement

```suggestion
```